src/client/components/molecules/portfolioSteps/education/index.tsx

```diff
@@ -29,27 +28,16 @@ const EducationForm = ({ changeRoute }: EducationProps) => {
 
   return (
     <div className="educationForm">

```

Comment:
even tho this was added in the past, can you please enforce the BEM pattern here?
classNames should be with `kebab-case` instead of `camelCase`

suggestin: `education-form` - `education-form__title` - `education-form__line`